src/lib/registration/registrationInitialisation.ts

```diff
@@ -109,7 +109,6 @@ export const initBeaconUse = (): BeaconUse => {
     windfarm: "",
     windfarmLocation: "",
     windfarmPeopleCount: "",
-    otherActivityDescription: "",
```

Comment:
This property was only used in mocks; no test/production code touched it, so I deleted it as unused.  `otherActivityText` property remains and is used.